engine/resource/src/resource_archive.cpp

```diff
@@ -59,7 +59,6 @@ namespace dmResourceArchive
     }
 
     void RegisterResourceDecryption(FDecryptResource decrypt_resource) {
-        assert(g_ResourceDecryption == 0);
```

Comment:
This was wrong. We assign a default decryption on init and this assert prevented anyone from changing to a custom one.